src/components/KuadrantTLSPage.tsx

```diff
@@ -0,0 +1,306 @@
+import * as React from 'react';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  FormSelectOption,
+  ActionGroup,
+  Button,
+  HelperText,
+  HelperTextItem,
+  FormHelperText,
+  FormSelect,
+  Page,
+  PageSection,
+  Title,
+  Radio,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import Helmet from 'react-helmet';
+import {
+  useK8sWatchResource, useK8sModel, getGroupVersionKindForResource, ResourceYAMLEditor
+} from '@openshift-console/dynamic-plugin-sdk';
+
+import './kuadrant.css';
+import { handleCreate } from '../utils/createResource';
+import { handleCancel } from '../utils/cancel';
+
+import { useHistory, Link } from 'react-router-dom';
+import NamespaceSelect from './namespace/NamespaceSelect';
+import yaml from 'js-yaml';
+
+
+
+const KuadrantTLSPage: React.FC = () => {
+
+  const history = useHistory();
+  const [policyName, setPolicyName] = React.useState('');
+  const [selectedNamespace, setSelectedNamespace] = React.useState("");
+  const [selectedGateway, setSelectedGateway] = React.useState("");
+  const [gateways, setGateways] = React.useState([]);
+  const [certIssuers, setCertIssuers] = React.useState([]);
+  const [selectedCertIssuer, setSelectedCertIssuer] = React.useState("");
+
+
+  //Initial skeleton setup for the yaml view
+  const skeletonTls = () => ({
+    apiVersion: 'kuadrant.io/v1alpha1',
+    kind: 'TLSPolicy',
+    metadata: {
+      name: '',
+      namespace: '',
+    },
+    spec: {
+      targetRef: {
+        group: 'gateway.networking.k8s.io',
+        kind: 'Gateway',
+        name: '',
+      },
+      issuerRef: {
+        name: selectedCertIssuer,
+        kind: '',
+      },
+    },
+  })
+  // Creates TLS policy object to be used for form and yaml creation of the resource
+  const createTlsPolicy = () => ({
+    apiVersion: 'kuadrant.io/v1alpha1',
+    kind: 'TLSPolicy',
+    metadata: {
+      name: policyName,
+      namespace: selectedNamespace,
+    },
+    spec: {
+      targetRef: {
+        group: 'gateway.networking.k8s.io',
+        kind: 'Gateway',
+        name: selectedGateway,
+      },
+      issuerRef: {
+        name: selectedCertIssuer,
+        kind: 'ClusterIssuer',
```

Comment:
I think these can also use NS scoped `Issuer`'s too?

Reply:
do you mean you can specify a NS for the issuer?

Reply:
No i mean cert-manager can use either a `ClusterIssuer` (cluster-scoped) or an `Issuer` (ns-scoped)

Reply:
oh i did not know there was two types makes sense ill change that